Client/Frontend/Home/HomeTabBanner.swift

```diff
@@ -0,0 +1,240 @@
+            cardView.leadingAnchor.constraint(greaterThanOrEqualTo: containerView.leadingAnchor, constant: 20),
+            cardView.trailingAnchor.constraint(lessThanOrEqualTo: containerView.trailingAnchor, constant: -20),
+            cardView.centerXAnchor.constraint(equalTo: containerView.centerXAnchor),
+            cardView.widthAnchor.constraint(equalToConstant: UX.cardSize.width),
+            cardView.heightAnchor.constraint(equalToConstant: UX.cardSize.height),
+            
+            image.topAnchor.constraint(equalTo: cardView.topAnchor, constant: 48),
+            image.leadingAnchor.constraint(equalTo: cardView.leadingAnchor, constant: 16),
+            image.widthAnchor.constraint(equalToConstant: UX.logoSize.width),
+            image.heightAnchor.constraint(equalToConstant: UX.logoSize.height),
+
+            bannerTitle.topAnchor.constraint(equalTo: image.topAnchor, constant: -16),
+            bannerTitle.leadingAnchor.constraint(equalTo: image.trailingAnchor, constant: 16),
+            bannerTitle.trailingAnchor.constraint(equalTo: cardView.trailingAnchor),
+
+            descriptionText.topAnchor.constraint(equalTo: bannerTitle.bottomAnchor),
+            descriptionText.leadingAnchor.constraint(equalTo: bannerTitle.leadingAnchor),
+            descriptionText.trailingAnchor.constraint(equalTo: cardView.trailingAnchor, constant: -12),
+            descriptionText.bottomAnchor.constraint(greaterThanOrEqualTo: CTAButton.topAnchor, constant: -16),
+
+            dismissButton.topAnchor.constraint(equalTo: cardView.topAnchor, constant: 16),
+            dismissButton.trailingAnchor.constraint(equalTo: cardView.trailingAnchor, constant: -16),
+            dismissButton.heightAnchor.constraint(equalToConstant: 16),
+            dismissButton.widthAnchor.constraint(equalToConstant: 16),
+
+            CTAButton.centerXAnchor.constraint(equalTo: cardView.centerXAnchor),
+            CTAButton.bottomAnchor.constraint(equalTo: cardView.bottomAnchor, constant: -16),
+            CTAButton.widthAnchor.constraint(equalToConstant: UX.learnHowButtonSize.width),
+            CTAButton.heightAnchor.constraint(equalToConstant: UX.learnHowButtonSize.height)
+        ])
+    }
+    
+    /// Apply message data, including handling of cases where certain parts of the message are missing.
+    private func applyMessage() {
+        
+        /// If no messages exist, continue using our evergreen message.
+        guard let message = message else {
+            bannerTitle.text = String.DefaultBrowserCardTitle
+            descriptionText.text = String.DefaultBrowserCardDescription
+            CTAButton.setTitle(String.DefaultBrowserCardButton, for: .normal)
+            
+            return
+        }
+        
+        guard let buttonLabel = message.data.buttonLabel else {
+            dismissClosure?()
+            
+            messagingManager.onMalformedMessage(messageKey: message.Id)
+            
+            return
+        }
```

Comment:
If the `buttonLabel` is nil, then the button should be elided, and the whole message should be clickable.